src/main/java/io/codelex/flightplanner/LogisticService.java

```diff
@@ -0,0 +1,121 @@
+package io.codelex.flightplanner;
+
+import io.codelex.flightplanner.Objects.Airport;
+import io.codelex.flightplanner.Objects.Flight;
+import io.codelex.flightplanner.Objects.FlightsRequest;
+import io.codelex.flightplanner.Objects.SearchItems;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+
+@Service
+public class LogisticService {
+    LogisticRepository logisticRepository;
+
+    public LogisticService(LogisticRepository logisticRepository) {
+        this.logisticRepository = logisticRepository;
+    }
+
+    public synchronized Flight addFlight(Flight flight) {
```

Comment:
Nu šī metode gan nav ok - Tev izskatās lasāmi un saprotami? :D Kā redzu pašam nācās rakstīt komentārus, lai to kodu vispār pats varētu uztvert.
Atver Spring prezentāciju un tur ir Validation slaidi, kuros pateikts kā salikt anotiācijas, lai nebūtu jāraskta šitā čupa.
Kad to izdarīsi, vēlreiz uzmetīšu aci.